compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -293,7 +293,7 @@ fn run_compiler(
     >,
     using_internal_features: Arc<std::sync::atomic::AtomicBool>,
 ) -> interface::Result<()> {
-    let mut early_error_handler = EarlyErrorHandler::new(ErrorOutputType::default());
+    let mut default_handler = EarlyErrorHandler::new(ErrorOutputType::default());
```

Comment:
I don't think the new name is any better. It is only supposed to be used for error handling very early on. After the Session is created it should't be used anymore.

Reply:
All uses of `let handler = EarlyErrorHandler::new(sess.opts.error_format);` have `Session` passed in as argument and thus should work with `sess.fatal("...");` just fine. I think it would make sense to remove all `EarlyErrorHandler` usage when `sess` is in scope and remove the `error_format` argument from `EarlyErrorHandler`.